src/renderer/component/publishForm/internal/bid-help-text.jsx

```diff
@@ -58,16 +59,35 @@ class BidHelpText extends React.PureComponent<Props> {
         </React.Fragment>
       );
     }
-
-    return winningBidForClaimUri ? (
+    
+    if (!winningBidForClaimUri && !winndingBidForClaimWithChannel) {
+      return __('Any amount will give you the winning bid.');
+    }
+    
+    const generateTakeOverMessage = (amount, uri) => {
+      return `${__('A deposit greater than')} ${amount} ${__('is needed to win')} ${uri}`
```

Comment:
Is "win" the right word here? Maybe control or mention vanity? We use it in a few places...

Reply:
I like control. I think we can definitely improve this copy.